.github/workflows/push.yml

```diff
@@ -435,7 +435,7 @@ jobs:
           CERT_CONTENTS: ${{ secrets.WINDOWS_CERT_PFX }}
         run: |
           base64 -d <<<"$CERT_CONTENTS" > ./cert.pfx
-          printf "cert_file=%s" ".\\cert.pfx" >> $GITHUB_OUTPUT
+          printf "::set-output name=cert_file::%s\n" ".\\cert.pfx"
```

Comment:
myslim, ze tu iba chyba `\n`, nie?

Reply:
(ked rozkliknes `run` v screenshote, tak tam by mali byt vsetky ENVs)

Reply:
Kde? 🤔 Na konci stringu v prvním parametru `printf` je `\n`.

Reply:
Jo vidíš no, v tom následujícím `sign` stepu je vidět že se certifikát nahrál správně. 🤔 Tý jo, tak já nevím, to bude možná úplně unrelated problém s tím buildem MSI. 🙄 

![CleanShot 2022-10-17 at 08 35 35@2x](https://user-images.githubusercontent.com/215660/196105663-f5ee6bd3-9e96-4d78-a6fa-dbf93401e774.jpg)

Reply:
Tak já to zkusím. 😅

Reply:
Myslel som, že v tom starom kóde chyba \n, že možno preto nejde.

Reply:
No ale ono to právě doteď fungovalo. ☹️